lib/modules/manager/mix/extract.spec.ts

```diff
@@ -15,27 +15,7 @@ describe('modules/manager/mix/extract', () => {
 
     it('extracts all dependencies', async () => {
       const res = await extractPackageFile(Fixtures.get('mix.exs'), 'mix.exs');
-      expect(res).toMatchSnapshot({
-        deps: [
-          { depName: 'postgrex', currentValue: '~> 0.8.1' },
-          { depName: 'ecto', currentValue: '>2.1.0 or <=3.0.0' },
-          {
-            depName: 'cowboy',
-            currentValue: 'ninenines/cowboy',
-            datasource: 'github',
-            skipReason: 'non-hex-dep-types',
-          },
-          {
-            depName: 'secret',
-            currentValue: '~> 1.0',
-            datasource: 'hex',
-            packageName: 'secret:acme',
-          },
-          { depName: 'ex_doc', currentValue: '>2.1.0 and <=3.0.0' },
-          { depName: 'jason', currentValue: '>= 1.0.0' },
-          { depName: 'jason', currentValue: '~> 1.0' },
-        ],
-      });
+      expect(res).toMatchSnapshot();
```

Comment:
revert and add missing matchers

Reply:
There's already a snapshot defined and there are no calculated properties to pass property matchers. Why should I revert and what matchers are missing?

Reply:
OK, makes sense!